models/webhook.go

```diff
@@ -650,6 +668,25 @@ func PrepareWebhook(w *Webhook, repo *Repository, event HookEventType, p api.Pay
 	return prepareWebhook(x, w, repo, event, p)
 }
 
+// getPayloadBranch returns branch for hook event, if applicable.
+func getPayloadBranch(p api.Payloader) (string, bool) {
```

Comment:
It would be better if this function return only string and treat empty string as not applicable

Reply:
Although branch name cannot be an empty string (actual rules are a bit more complicated than that), don't you think that `ok` flag better conveys intention? If Go had optional types, I think we'd certainly use them here instead of having empty string having an implicit `None` value. Since Go doesn't have them, `value, ok` pattern is the best we can get here (which is widely used, both in builin types and libraries).

Reply:
In some cases yes but there is no way this function can return non-empty string and  false for ok at the same time